lua/lualine/components/signify.lua

```diff
@@ -1,9 +1,90 @@
 -- Copyright (c) 2020-2021 shadmansaleh
 -- MIT license, see LICENSE for more details.
 
+local async = require'lualine.async'
```

Comment:
it would be nice to implement this without the async module. No need to have 75sloc just to call a single command here.

Reply:
I think we should keep the async module . This will make it easy to write other components that rely on extarnal program on future. Also I donn't think this module will require much maintanace as it already does basicaly all it needs to do . 75loc doesn't seem too much to me.

Reply:
Yea my concern is that I don't see another use case for this module for now. But I agree that async module is really simple so we can keep it. I will review and hopefully merge the rest of this pr later today.